test/app/package.json

```diff
@@ -5,7 +5,8 @@
     "dev": "vite",
     "build": "vite build",
     "serve": "vite preview",
-    "test": "node ../../cli"
+    "test": "node ../../cli",
```

Comment:
Could you combine the two tests, so:
`"test": "node ../../cli && node ../../cli -e test-flags.html -p 3006"`
Then we will be testing both default values and argument passing during CI.